control/pid.go

```diff
@@ -125,41 +233,95 @@ func (p *basicPID) reset() error {
 		p.limLo = p.cfg.Attribute["limit_lo"].(float64)
 	}
 
-	p.tuning = false
-	if p.kI == 0.0 && p.kD == 0.0 && p.kP == 0.0 {
-		var ssrVal float64
-		if p.cfg.Attribute["tune_ssr_value"] != nil {
-			ssrVal = p.cfg.Attribute["tune_ssr_value"].(float64)
-		}
+	if p.useMulti {
+		for i := 0; i < len(p.PIDSets); i++ {
+			if p.PIDSets[i].NeedsAutoTuning() {
+				var ssrVal float64
+				if p.cfg.Attribute["tune_ssr_value"] != nil {
```

Comment:
so in this case all pid's in a MIMO block need to use the same values? (excluding P, I, and D). otherwise how would it know which "tune_ssr_value" to use?

Reply:
jk I see it in a comment below

Reply:
optional nit to move that comment from line 254 to line ~239, but nbd